src/js/utils/breadcrumb-helper.js

```diff
@@ -0,0 +1,169 @@
+const config = {
```

Comment:
I'm a little concerned that we're doing this much direct DOM manipulation in a React component. I realize you're reusing code that's also being used for the static pages that isn't in React, but given that we're already going to have maintain two slightly different versions of this code, I wonder if it might be better to have a React breadcrumbs component that is written in idiomatic React and use that in the React apps that need it. I'm pretty sure the React code for this would much simpler than the DOM version here.

Reply:
☝️ ➕